packages/system-tests/scenarios/forceSoqlQueryUi.test.ts

```diff
@@ -27,17 +27,22 @@ describe(TITLE, () => {
 
   let username: string;
 
-  before(async () => {
+  beforeEach(async () => {
```

Comment:
Updating from `before` to `beforeEach` and `after` to `afterEach` causes the tests to run in independent vscode sessions instead of sharing the same vscode session. This is important because the tests themselves were only asserting the presence of `exit code 0` in the output panel. If the tests share the same vscode session, as long as one of the tests is successful then the rest would pass even if their command execution fails.